src/lib/chips/chip/chip-foundation.ts

```diff
@@ -8,7 +8,7 @@ export interface IChipFoundation extends ICustomElementFoundation {
   selected: boolean;
   disabled: boolean;
   invalid: boolean;
-  value: string;
+  value: any;

```

Comment:
Not that this should be addressed here and now but have we given any thought to phasing out `any` and replacing it with `unknown` throughout the library?

Reply:
I'd be all in favor of that, and almost used it for this one but since it was already `any` in some places didn't want to rock the boat with this PR.  With the introduction of `unknown` there are very few places `any` is actually needed anymore.